.setup/bin/refresh_vagrant_workers.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+import pwd
+import json
+import subprocess
+from collections import OrderedDict
+
+SUBMITTY_REPOSITORY = os.environ['SUBMITTY_REPOSITORY']
```

Comment:
these environment variables don't exist on production, let's not use this here...